dotnet/private/rules/common/attrs.bzl

```diff
@@ -130,6 +130,11 @@ COMMON_ATTRS = {
         doc = "Whether or not to override the strict_deps attribute.",
         mandatory = False,
     ),
+    "allow_unsafe": attr.bool(
```

Comment:
Could you move this to `CSHARP_COMMON_ATTRS` instead since `unsafe` is not available in F#